CHANGELOG.md

```diff
@@ -1,5 +1,97 @@
 # Changelog
 
+## v1.8.5
+
+Summary of Changes
+------------------
+
+**Minor Changes:**
+* Add a new daemon CLI argument, "--iptables-random-fully" to specify the
 iptables "--random-fully" argument when invoking the iptables CLI binary
 directly from cilium-agent. (Backport PR #13700, Upstream PR #13383, @kh34)
```

Comment:
The newlines (sometimes rendered as ^M) might show up a bit weird here and/or in the release notes. Maybe we should fix those before merge?

Reply:
Good catch. Fixed